src/index.html

```diff
@@ -8,10 +9,52 @@
 
   <link rel="stylesheet" href="css/main.min.css" />
 </head>
+
 <body>
-  <h1>Hello, World!</h1>
-  <h2>Yarik lox</h2>
+  <header>
```

Comment:
Add class to header tag

Reply:
I don`t have this lines (12 and 13) in my htlm file

Reply:
Added